src/test/resources/features/Social/SpacesWidget.feature

```diff
@@ -5,6 +5,15 @@ Feature: Spaces widget checking
   I want to check the spaces in home page
   In order to validate the page
 
+  @smoke
+  Scenario: US 3.3.2_(01) [BACK] No Spaces requests to join
+    Given I am authenticated as 'admin' random user
+    And I create the tenth random user if not existing
```

Comment:
the tenth user does not exist in TestInitHook, can you choose an existing one ? 
    String[] randomUsers = new String[] {
        "first",
        "second",
        "third",
        "fourth",
        "fifth",
        "sixth",